src/ngx_http_lua_kong_module.c

```diff
@@ -117,6 +118,18 @@ ngx_http_lua_kong_get_module_ctx(ngx_http_request_t *r)
     return ctx;
 }
 
+static void *
+ngx_http_lua_kong_create_main_conf(ngx_conf_t* cf)
+{
+    ngx_http_lua_kong_main_conf_t *conf;
+
+    conf = ngx_pcalloc(cf->pool, sizeof(ngx_http_lua_kong_main_conf_t));
+    if (conf == NULL) {
+        return NULL;
+    }
+
+    return conf;
+}
```

Comment:
we are deleting the `t/008-log.t` tests because we need the `nginx` patch, right? can we have some way of having the patch here? it worries me a bit that we are deleting the tests, in fact I wanted to add more tests with this change...